### ISSUE TITLE
Error if CWD is not node_modules' parent directory

### ISSUE THREAD

**Issue by rcarton** [ahmadnassri/nightwatch-accessibility#4]
```
{ Error: ENOENT: no such file or directory, open '/app/test/ui/node_modules/axe-core/axe.min.js'
    at Object.fs.openSync (fs.js:653:18)
    at Object.fs.readFileSync (fs.js:554:33)
```
Nightwatch is running from `/app/test/ui`, but `node_modules` is in `/app/node_modules`.

You could use https://www.npmjs.com/package/find-node-modules here instead
```
const source = fs.readFileSync(path.resolve(path.join('node_modules', 'axe-core', 'axe.min.js')), 'utf8')```

**Comment by mmoss** [comment]
https://github.com/ahmadnassri/nightwatch-accessibility/pull/16 resolves this without having to use any other dependencies.

**Comment by ahmadnassri** [comment]
:tada: This is included in version 1.8.0 :tada:

The release is available on:
- [GitHub release](https://github.com/ahmadnassri/nightwatch-accessibility/releases/tag/v1.8.0)
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/nightwatch-accessibility)
